database/prod/queryClauses/baseClause.cpp

```diff
@@ -0,0 +1,15 @@
+#include "baseClause.h"
+
+QString BaseClause::text() const {
+    return QString("\n %1 %2").arg(title(), valueText());
+}
+
+QString BaseClause::title() const {
+    return title_;
+}
+
+BaseClause::~BaseClause() {}
+
+BaseClause::BaseClause(const QString& title)
+    : title_(title)
+{};
```

Comment:
а зачем точка с запятой в конце?
```suggestion
{
}
```